tests/fixtures/models/model-relationships-morphone-morphmany-laravel8.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Flag extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'user_id',
+    ];
+
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'id' => 'integer',
+        'user_id' => 'integer',
+    ];
+
+
+    public function order()
```

Comment:
Shouldn't this method name be `stars`?

Reply:
Totally, just refactor! that was my question about it! 

Just pushed the fix! 

Thanks 👍